src/tests/ftest/soak/soak.yaml

```diff
@@ -46,154 +76,134 @@ container_reserved:
-# smoke job params; Single job; one task per node; np = # nodes
-jobsmoke:
-    name: job_smoke
-    time: 10
-    tasks:
-        - 1
-    jobspec:
-        - smoke_ior_daos
-        - smoke_ior_mpiio
-        # - dmg_create_destroy
-# IOR daos job params; multiple jobs, one job per node;
-# 1, 16 and 32 tasks per node
-job_ior_daos:
-    name: job_ior_daos
-    time: 20
-    tasks:
-        - 1
-        - 16
-        #- 32
-    jobspec:
-        - ior_daos
-# IOR MPIIO job params; multiple jobs, one job per node;
-# 1, 16 and 32 tasks per node
-job_ior_mpiio:
-    name: job_ior_mpiio
-    time: 20
-    tasks:
-        - 1
-        - 16
-        #- 32
-    jobspec:
-        - ior_mpiio
-# TBD job params for DMG
-job_dmg_create_destroy:
-    name: job_dmg_create_destroy
-    time: 20
-    jobspec:
-        - dmg_create_destroy
-# Commandline parmeters - job specs
+      - ior_stress
+      - fio_stress
+      #- dmg_create_destroy
+      #- dfs
+# Commandline parameters
 # Benchmark and application params
 # IOR params -a DAOS and -a MPIIO
 # sequential
-smoke_ior_daos:
-    api: DAOS
+ior_smoke:
+    api:
+        - DAOS
+        - MPIIO
     test_file: daos:testFile
-    flags: -v -w -W -r -R
+    flags: -v -w -W -r -R -q
     block_size:
         - '64M'
-    repetition: 1
+    repetitions: 1
     transfer_size:
         - '4k'
         - '128k'
         - '1m'
+    segment_count: 1
     daos_oclass:
         - 'SX'
-smoke_ior_mpiio:
-    api: MPIIO
+fio_smoke:
+  names:
+    - global
+    - test
+  global:
+    directory: "/tmp/daos_dfuse"
+    ioengine: 'libaio'
+    thread: 1
+    group_reporting: 1
+    direct: 1
+    verify: 'crc64'
+    iodepth: 16
+  test:
+    numjobs: 1
+  soak:
+    blocksize:
+      - '1M'
+    size:
+      - '1G'
+    rw:
+      - 'rw'
+      - 'randrw'
+ior_stress:
+    api:
+        - DAOS
+        - MPIIO
     test_file: daos:testFile
-    flags: -v -w -W -r -R
+    flags: -v -w -W -r -R -q
     block_size:
         - '64M'
-    repetition: 1
+        - '4M'
+        - '32M'
+    repetitions: 5
     transfer_size:
         - '4k'
         - '128k'
         - '1m'
+        - '512K'
+        - '64k'
     daos_oclass:
-        - 'SX'
-ior_daos:
-    api: DAOS
-    test_file: daos:testFile
-    flags: -v -w -W -r -R
-    block_size:
-        - '64M'
-    repetition: 5
-    transfer_size:
-        - '4k'
-        - '512k'
-        - '1m'
-    daos_oclass:
-        - 'SX'
-        #- 'RP_G3X'
-ior_mpiio:
-    api: MPIIO
-    test_file: daos:testFile
-    flags: -v -w -W -r -R
-    block_size:
-        - '64M'
-    repetition: 5
-    transfer_size:
-        - '4k'
-        - '512k'
-        - '1m'
-    daos_oclass:
-        - 'SX'
-        #- 'RP_G2X'
-# dmg cmdline params - TBD
+        - "SX"
+        # - "RP_2GX"
+        # - "RP_3GX"
+        # - "RP_4GX"
+fio_stress:
+  names:
+    - global
+    - test
+  global:
+    directory: "/tmp/daos_dfuse"
+    ioengine: 'libaio'
+    thread: 16
```

Comment:
This needs to be 1. thread param is a bool. It can either be 1 or 0. Meaning it will either use threads or processes.

Reply:
changed thread: 1